model/daal4py-LR/daal4py_LR/daal4py.py

```diff
@@ -0,0 +1,95 @@
+import pathlib
+import statistics
+import importlib
+from typing import AsyncIterator, Tuple, Any, Type, List
+
+from dffml import (
+    config,
+    field,
+    entrypoint,
+    SimpleModel,
+    ModelNotTrained,
+    Accuracy,
+    Feature,
+    Features,
+    Sources,
+    Record,
+)
+
+
+@config
+class daal4pyModelConfig:
+    predict: Feature = field("Label or the value to be predicted")
+    features: Features = field("Features to train on. For SLR only 1 allowed")
+    directory: pathlib.Path = field(
+        "Directory where state should be saved",
+        default=pathlib.Path("~", ".cache", "dffml", "daal4py"),
+    )
+
+class daal4pyModelContext(SimpleModelContext):
```

Comment:
```suggestion
@entrypoint("daal4pylr")
class DAAL4PyLRModel(SimpleModel):
```

Reply:
With `SimpleModel` you only need one class, rather than needed two classes (the Base + Context pair)